src/main/steps/c100-rebuild/other-proceedings/documentUpload/postController.ts

```diff
@@ -133,23 +150,34 @@ export default class UploadDocumentController extends PostController<AnyObject>
   private buildOrderTypeName(courtOrderType: C100OrderTypes) {
     return C100OrderTypeNameMapper[courtOrderType].split(' ').join('_').toLowerCase();
   }
-
+  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+  public isValidFileFormat = (documents: any): boolean => {
```

Comment:
@kumarmadurai Can you please add test cases for these new validations file extension & file max size ?